src/dbmanager.cpp

```diff
@@ -1019,7 +1019,7 @@ NodeData DBManager::getNode(int nodeId)
                            R"("title" )"
                            R"(FROM node_table WHERE id=:id LIMIT 1;)");
             query2.bindValue(":id", node.parentId());
-            bool status = query2.exec();
+            status = query2.exec();
```

Comment:
Do we need `status` at all? Why would we not just write `if (query2.exec()) {`?

Reply:
Good point! But I don't think I should update *all* instances of `status = query2.exec()` in this PR (there are around 80 of them!). It seems a little out of scope to me...